#!/usr/bin/env python
PACKAGE = "object_vision"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("downsample_size", double_t, 0, "Downsampling Leaf Size",    .02, 0,   0.03)
gen.add("min_angle", double_t, 0, "Minimum angle of plane (cus dps only)",    0, 0,   3.14)
gen.add("max_angle", double_t, 0, "Maximum angle of plane (cus dps only)",    1.57, 0,   3.14)
gen.add("cluster_tolerance", double_t, 0, "Distance between clusters",    0.5, 0,   0.25)
gen.add("min_cluster_size", int_t, 0, "Object Seg Minimum Points in a Cluster",    20, 5, 100)
gen.add("object_min_height", double_t, 0, "Set minimum object height above plane.",    0.02, 0,   0.5)
gen.add("object_max_height", double_t, 0, "Set maximum object height above plane.",    0.5, 0,   1.0)
gen.add("maximum_distance_thresh", double_t, 0, "Set maximum object height above plane.",    1.5, 0,   2.0)



exit(gen.generate(PACKAGE, "dynamic_vision", "segmentation"))
